hack/get_ci_vm.sh

```diff
@@ -90,17 +82,31 @@ show_usage() {
 }
 
 get_env_vars() {
-    python -c '
-import yaml
+    # Deal with both YAML and embedded shell-like substitutions in values
+    # if substitution fails, fall back to printing nakid env. var as-is.
+    python3 -c '
+import yaml,re
 env=yaml.load(open(".cirrus.yml"), Loader=yaml.SafeLoader)["env"]
-keys=[k for k in env if "ENCRYPTED" not in str(env[k])]
+rx1=re.compile(r"\$(\w+)")
+rx2=re.compile(r"\$\{(\w+)\}")
+rep=r"{\1}"
```

Comment:
This is really clever - so clever that it took me a long while to figure it out. Would you consider adding comments to help the next maintainer?

Reply:
yep, you got it boss.